# KitObject message
# This structure contains the information of an object contained in a kit.

# Object type
string type

# Target pose
geometry_msgs/Pose pose
